### ISSUE TITLE
Keep last pie chart segment hovered

### ISSUE THREAD

**Issue by jeffhuys** [novus/nvd3#1432]
When I hover my mouse over a segment in my pie chart, it "lights up" and displays some information next to it. I want the segment to stay "lit" until another one is hovered over, even when the user navigates different sections of the page.

This is the code I use to display the pie:

```
		$scope.options = {
			chart: {
				pie: {
					dispatch: {
						elementMouseover: function(element) {
							$scope.$apply(function(scope) {
								scope.donutMiddleText = element.data.key; 
								scope.benefitText = element.data.text;
								console.log(element);
							});
						}
					}
				},
				type: 'pieChart',
				height: 300,
				width: 300,
				x: function(d){return d.key;},
				y: function(d){return d.y;},
				showLabels: false,
				donut: true,
				color: ['rgba(0,0,0,0.15)'],
				showLegend: false,
				duration: 500,
				labelThreshold: 0.01,
				labelSunbeamLayout: true,
				tooltips: false,
				legend: {
					margin: {
						top: 0,
						right: 0,
						bottom: 0,
						left: 0
					}
				}
			}
		};
```

Any help is appreciated!

**Comment by jeffhuys** [comment]
Please? 🙊

**Comment by liquidpele** [comment]
I could see keeping it highlighted after clicking, but just on hover would be odd, you'd never be able to see the non-highlighted view again unless you refreshed the whole page.

**Comment by jeffhuys** [comment]
Yeah, the thing is, we're using it for an unusual use case. We use it as a donut graphic which is tied to a set of instruction (6 steps). We want the first one to be highlighted when the page loads, and on hover, show a different text. This already works, but not the "keeping it enabled" part.

**Comment by liquidpele** [comment]
Instead of highlighting, you could tell it to make that one have a higher radius so that it stands out... kind of like what's shown here:

https://nvd3-community.github.io/nvd3/examples/monitoringChart.html

**Comment by jeffhuys** [comment]
Hmm, thanks! That may be a nice solution. Weird, though, that there's no way to color a segment.
